env.sh

```diff
@@ -56,20 +45,16 @@ if [ $? -ne 0 ]; then
     docker run -d \
            --net=${SYSTEM_NET} \
            --volume ${SOURCE_DIR}:${WORKDIR} \
-           --volume ${DOCKER_CARGO}/git:${DOCKER_CARGO}/git \
```

Comment:
This is a buffer for patch from crate.io. Do not delete this.

Reply:
Okay, I got it.